layouts/community/ortho_4x12/mindsound/flicker.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FLICKER_H
```

Comment:
Could you replace the ifndef and the define here with just `#pragma once`, and remove the endif at the end?

Reply:
aye aye